internal/nodes/avi_model_l4_translator.go

```diff
@@ -441,10 +441,17 @@ func getAutoFQDNForService(svcNamespace, svcName string) string {
 	if lib.GetL4FqdnFormat() == lib.AutoFQDNDefault {
 		// Generate the FQDN based on the logic: <svc_name>.<namespace>.<sub-domain>
 		fqdn = svcName + "." + svcNamespace + "." + subdomain
+
 	} else if lib.GetL4FqdnFormat() == lib.AutoFQDNFlat {
 		// Generate the FQDN based on the logic: <svc_name>-<namespace>.<sub-domain>
 		fqdn = svcName + "-" + svcNamespace + "." + subdomain
 	}
+	if !lib.CheckRFC1035(fqdn) {
```

Comment:
The fqdn can be auto-generated from within AKO itself, which is not an explicit user input, in which case 
1. either this should not be enforced.
2. the auto-generated fqdn must be compliant at all times regardless of any user input.

Reply:
We need to follow the label standard RFC 1035. Either we can ask them to provide the svcName starting with an alphabetic character or we can add that tag "svc" when they don't. The other alternative is to always add the tag. Please suggest.